src/pt/yomucomics/build.gradle

```
@@ -1,9 +1,9 @@
 ext {
-    extName = 'SSSScanlator'
-    extClass = '.SSSScanlator'
+    extName = 'YomuComics'
```

Add a space between the words. In ```YomuComics::name```, you added a space.

```suggestion
    extName = 'Yomu Comics'
```

---

Fix it